https://bugs.launchpad.net/panotools/+bug/686499

ColourBrightness.c: In function 'OutputPhotoshopCurve':
ColourBrightness.c:412:3: warning: implicit declaration of function 'htons'

The fix neesds to be done better (check whether arpa/inet.h is available) to be upstreamable.

Index: ColourBrightness.c
===================================================================
--- ColourBrightness.c.orig
+++ ColourBrightness.c
@@ -35,6 +35,7 @@
 #include "tiffio.h"
 #include <assert.h>
 #include <math.h>
+#include <arpa/inet.h>
 
 #include "pttiff.h"
 
